try {
    console.log(dddd);
} catch (error) {
    console.log(error);
} finally {
    console.log("код, который будет выполнен в любом случае");
}
